source/fundamentals/crud/write-operations/bulk-write.txt

```diff
@@ -0,0 +1,546 @@
+bulk write operation:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Property
+     - Description
+
+   * - ``BypassDocumentValidation``
+     - | Specifies whether the operation bypasses document-level validation. For more
+         information, see :manual:`Schema
+         Validation </core/schema-validation/#schema-validation>` in the {+mdb-server+}
+         manual.
+       | Defaults to ``false``.
+
+   * - ``Comment``
+     - | A comment to attach to the operation, in the form of a ``BsonValue``. For
+         more information, see the :manual:`delete command
+         fields </reference/command/delete/#command-fields>` guide in the
+         {+mdb-server+} manual.
+
+   * - ``IsOrdered``
+     - | If ``true``, the driver performs the write operations in the order
+         provided. If an error occurs, the remaining operations are not
+         attempted. 
+       |
+       | If ``false``, the driver performs the operations in an
+         arbitrary order and attempts to perform all operations. If any of the write
+         operations in an unordered bulk write fail, the driver
+         reports the errors only after attempting all operations.
+       | Defaults to ``True``.
+
+   * - ``Let``
+     - | A map of parameter names and values, in the form of a ``BsonDocument``. Values
+         must be constant or closed
+         expressions that don't reference document fields. For more information,
+         see the :manual:`let statement
+         </reference/command/delete/#std-label-delete-let-syntax>` in the
+         {+mdb-server+} manual.
+    
+   * - ``VerboseResult``
+     - | Specifies whether the ``ClientBulkWriteResult`` object returned by the operation
+         includes detailed results for each successful write operation.
+       | Defaults to ``false``.
+
+   * - ``WriteConcern``
+     - | The write concern to use for the write operation, as a value from the ``WriteConcern``
+         enum.
+       | Defaults to the write concern of the collection on which the operation is running.
+
+The following code examples use a ``ClientBulkWriteOptions`` object to customize
+a delete operation:
+
+.. tabs::
+
+   .. tab:: Asynchronous
+      :tabid: bulk-write-options-async
+
+      .. literalinclude:: /includes/fundamentals/code-examples/BulkWrite.cs
+         :start-after: start-bulk-write-options-async
+         :end-before: end-bulk-write-options-async
+         :language: csharp
+         :copyable:
+         :dedent: 8
+
+   .. tab:: Synchronous
+      :tabid: bulk-write-options-sync
+
+      .. literalinclude:: /includes/fundamentals/code-examples/BulkWrite.cs
+        :start-after: start-bulk-write-options-sync
+        :end-before: end-bulk-write-options-sync
+        :language: csharp
+        :copyable:
+        :dedent: 8
+
+Return Value
+------------
+
+The ``BulkWrite()`` and ``BulkWriteAsync()`` methods return a ``ClientBulkWriteResult``
+object that contains the following properties:
```

Comment:
Should we describe `ClientBulkWriteException` so users know how to find error responses? Based on the spec: https://github.com/mongodb/specifications/blob/master/source/crud/bulk-write.md#exception